Sources/Public/CalendarView.swift

```diff
@@ -258,6 +258,10 @@ public final class CalendarView: UIView {
       invalidateIntrinsicContentSize()
     }
 
+    // Clear this so that we don't return early in our next layout pass even though our layout
+    // region might not have changed.
+    layoutRegion = nil
```

Comment:
we clear this in `setContent`, since we want `updateVisibleViews(...)` to run on the next layout pass even if our layout region hasn't changed